src/api/main.py

```diff
@@ -392,80 +476,84 @@ def get_sources(
-    if shot_id is not None:
-        query = query.where(models.SignalModel.shot_id == shot_id)
-    stream = ndjson_stream_query(db, query)
-    return StreamingResponse(stream, media_type="application/x-ndjson")
-
-
-@app.get(
-    "/ndjson/shots",
-    description="Get data on shots as an ndjson stream",
-)
-def get_shots_stream(
-    db: Session = Depends(get_db), params: QueryParams = Depends()
-) -> models.ShotModel:
-    query = crud.select_query(
-        models.ShotModel, params.fields, params.filters, params.sort
-    )
-    stream = ndjson_stream_query(db, query)
-    return StreamingResponse(stream, media_type="application/x-ndjson")
-
-
-@app.get(
-    "/ndjson/sources",
-    description="Get data on sources as an ndjson stream",
-)
-def get_sources_stream(
-    db: Session = Depends(get_db), params: QueryParams = Depends()
-) -> models.SourceModel:
-    query = crud.select_query(
-        models.SourceModel, params.fields, params.filters, params.sort
-    )
-    stream = ndjson_stream_query(db, query)
-    return StreamingResponse(stream, media_type="application/x-ndjson")
+# @app.get(
+#     "/ndjson/signals",
+#     description="Get data on signals as an ndjson stream",
+# )
+# def get_signals_stream(
+#     name: Optional[str] = None,
+#     shot_id: Optional[int] = None,
+#     db: Session = Depends(get_db),
+#     params: QueryParams = Depends(),
+# ) -> models.SignalModel:
+#     query = crud.select_query(
+#         models.SignalModel, params.fields, params.filters, params.sort
+#     )
+#     if name is None and shot_id is None:
+#         raise HTTPException(
+#             status_code=400, detail="Must provide one of a shot_id or a signal name."
+#         )
+#     if name is not None:
+#         query = query.where(models.SignalModel.name == name)
+#     if shot_id is not None:
+#         query = query.where(models.SignalModel.shot_id == shot_id)
+#     stream = ndjson_stream_query(db, query)
+#     return StreamingResponse(stream, media_type="application/x-ndjson")
+
+
+# @app.get(
+#     "/ndjson/shots",
+#     description="Get data on shots as an ndjson stream",
+# )
+# def get_shots_stream(
+#     db: Session = Depends(get_db), params: QueryParams = Depends()
+# ) -> models.ShotModel:
+#     query = crud.select_query(
+#         models.ShotModel, params.fields, params.filters, params.sort
+#     )
+#     stream = ndjson_stream_query(db, query)
+#     return StreamingResponse(stream, media_type="application/x-ndjson")
+
+
+# @app.get(
+#     "/ndjson/sources",
+#     description="Get data on sources as an ndjson stream",
+# )
+# def get_sources_stream(
+#     db: Session = Depends(get_db), params: QueryParams = Depends()
+# ) -> models.SourceModel:
+#     query = crud.select_query(
+#         models.SourceModel, params.fields, params.filters, params.sort
+#     )
+#     stream = ndjson_stream_query(db, query)
+#     return StreamingResponse(stream, media_type="application/x-ndjson")
```

Comment:
Just remove these, rather than commenting code out. 

The code is in git. If we ever need it again, one can go back in time and get it.